googleapiclient/errors.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, reason, resp=None, content=None):
     self.reason = reason
 
   def __repr__(self):
+    if self.resp is not None and hasattr(self.resp, 'status'):
```

Comment:
Why check the `None`-ness of `self.resp`? Why not just do the `hasattr` (or `getattr`) call?